### ISSUE TITLE
Is it possible to play youtube video using chewie?

### ISSUE THREAD

**Issue by yousifAlneamy** [brianegan/chewie#268]

**Comment by shashikantshaale** [comment]
No. Only network(with streaming url), assets or file videos can be played.

**Comment by shashikantshaale** [comment]
Checkout https://pub.dev/packages/youtube_player_flutter

**Comment by yousifAlneamy** [comment]
> Checkout https://pub.dev/packages/youtube_player_flutter

I tried it before but sometimes it shows youtube error codes on real devices. `flutter_youtube` is the most reliable one, but with limitations.
